### PR TITLE
Unable to save password in IntelliJ password database #138

### PR DESCRIPTION
**Pull request by speechkey**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -13,10 +13,10 @@ repositories {
     mavenCentral()
 }
 
-version = '2.0.12-SNAPSHOT'
+version = '2.0.13-SNAPSHOT'
 
 intellij {
-    version 'IC-143.381.42'
+    version 'IC-141.1532.4'
```
Comment (omayevskiy) [inline build.gradle]: what's the reason change the version?

**Comment by omayevskiy** [comment]
not reproducible

**Comment by speechkey** [comment]
Hi Oleg! Sorry for the late answer. The purpose of the line is only to enable the plugin on the earlier version of IntelliJ, which we are currently using in our team. The bug can certainly occur only on earlier versions of IntelliJ, if it was not tested/not supported on the earlier versions of IntelliJ. Which version of IntelliJ are currently supported?

**Comment by omayevskiy** [comment]
The plugin is normally developed against latest available version, but I try to keep older versions working as well but not forever.

**Comment by speechkey** [comment]
Ok. So the bug occurs under IntelliJ 13.

**Comment by omayevskiy** [comment]
ok for under 13 there is no support (well there is no real support at all).

**Pull request closed without merging**